docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "f8b7f2dc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture \n",
+    "!pip install cleanlab termcolor "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "33d67688",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# package versions: \n",
+    "# cleanlab==2.0.0 \n",
+    "# termcolor==1.1.0 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 2 \n",
+    "\n",
+    "In this tutorial, we show how you can use cleanlab to find potential label errors in token classification dataset. Here, we use CONLL-2003 which contains 3,449 sentences and 46,400 tokens (after filtering). \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Identify potential token label issues using cleanlab's `find_label_issues` method. \n",
+    "- Rank sentences using cleanlab's `token_classification.rank.get_label_quality_score` method. \n",
+    "- TODO: Train a more robust model by removing problematic sentences. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1da020bc",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a1349304",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from termcolor import colored \n",
+    "from cleanlab.filter import find_label_issues \n",
+    "from rank import get_label_quality_scores \n",
+    "from utils import * \n",
+    "# change to this after token_classification.rank is pushed to the official package \n",
+    "# from cleanlab.token_classification.rank import get_label_quality_scores "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ad75b45",
+   "metadata": {},
+   "source": [
+    "## 2. Get `pred_probs` and `labels` \n",
+    "\n",
+    "For more information on how to get `pred_probs` and `labels`, see part 1. Recall that we also need the number of word-level tokens for each sentence. Also get their nested list format. "
```

Comment:
Try to minimize references to part 1 as much as possible. Let's assume most users will never read part 1 of the tutorial (eventually I will even change the naming of these tutorials such that this is simply the "Token Classification" tutorial, and what you're currently calling "Part 1" is simply an auxiliary notebook that demonstrates how to train model to get pred_probs.

So this notebook also needs to explain the structure of `pred_probs` and `labels`.

Reply:
In particular, this notebook also should more clearly distinguish what the indices represent.
There should be some discussion about what is a "sentence" (could also be a "document") which are the first indices of `labels` and `pred_probs`, and what indices correspond to a "token within a sentence".